.eslintrc.json

```diff
@@ -17,30 +26,67 @@
           ],
           // Packages. `react` related packages come first.
           // Also, put `react-icons` in sorting order not with `react`
-          ["^react(?!-.)$", "^\\w", "^@\\w"],
+          [
+            "^react(?!-.)$",
+            "^\\w",
+            "^@\\w"
+          ],
           // The Chakra theme directory if imported to story file or other places
-          ["^@/@chakra-ui"],
+          [
+            "^@/@chakra-ui"
+          ],
           // From the `types` directory.
-          ["^@/lib/types", "^@/lib/interfaces"],
+          [
+            "^@/lib/types",
+            "^@/lib/interfaces"
+          ],
           // From the `components` directory.
-          ["^@/components"],
+          [
+            "^@/components"
+          ],
           // From the `utils` directory.
-          ["^@/lib/utils"],
+          [
+            "^@/lib/utils"
+          ],
           // From the `data` directory.
-          ["^@/data"],
+          [
+            "^@/data"
+          ],
           // From the `constants` directory.
-          ["^@/lib/constants"],
+          [
+            "^@/lib/constants"
+          ],
           // Parent imports. Put `..` last.
-          ["^\\.\\.(?!/?$)", "^\\.\\./?$"],
+          [
+            "^\\.\\.(?!/?$)",
+            "^\\.\\./?$"
+          ],
           // Other relative imports. Put same-folder imports and `.` last.
-          ["^\\./(?=.*/)(?!/?$)", "^\\.(?!/?$)", "^\\./?$"],
+          [
+            "^\\./(?=.*/)(?!/?$)",
+            "^\\.(?!/?$)",
+            "^\\./?$"
+          ],
           // Style imports.
-          ["^.+\\.s?css$"],
+          [
+            "^.+\\.s?css$"
+          ],
           // Side effect imports.
-          ["^\\u0000"]
+          [
+            "^\\u0000"
+          ]
         ]
       }
     ],
-    "simple-import-sort/exports": "error"
+    "simple-import-sort/exports": "error",
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/no-explicit-any": [
+      "error",
+      {
+        "fixToUnknown": true
+      }
+    ],
+    "unused-imports/no-unused-imports-ts": 2
```

Comment:
Same here, what does this one mean?

Reply:
Yeah, a warning is ok. As long as it is fixed by the `lint:fix` it is the same to me.